backend/helpers/helpers.go

```diff
@@ -72,9 +73,27 @@ func GetFirstTable[T models.AllTables, P models.AllTables](w http.ResponseWriter
 
 }
 
+func UpdateSpecifiedParams[T models.AllTables](w http.ResponseWriter, r *http.Request, table *T, initCommon models.Common, updatedCommon *models.Common) (error, T) {
+	err := json.NewDecoder(r.Body).Decode(&table)
+	if err != nil {
+		http.Error(w, "400 malformed request", http.StatusBadRequest)
```

Comment:
I don't think the status code should be included in the error message.

Reply:
if @AdrianJendo cares I'll update them all